vpr/src/route/router_lookahead_extended_map.h

```diff
@@ -19,6 +19,9 @@ class ExtendedMapLookahead : public RouterLookahead {
     ///<Look-up table from SOURCE/OPIN to CHANX/CHANY of various types
     util::t_src_opin_delays src_opin_delays;
 
+    ///< Lookup table from SOURCE/OPIN to CHANX/CHANY of the another layer
```

Comment:
In the update to the router lookahead (to support inter-layer connection from SW block) this data structure is deleted since there will be no use to it.